sites/topomath.asu.edu/modules/system_descriptions/system_descriptions.module

```diff
@@ -70,6 +72,7 @@ function createSpecificBlocks(){
 							</div>';
 	return $specific_blocks;
 }
+
```

Comment:
yes, as I told in the comment. This is part of modals (there are also few in nonClassUpdates ) which are similar in nature and as far as I know better to make all of this modal creation dynamic in separate module and should be a separate task. as far as rest of deleted html snippets are considered I made those blocks via admin panel